src/plugins/intel_cpu/src/utils/shape_inference/static_shape.hpp

```diff
@@ -17,7 +17,9 @@
 namespace ov {
 namespace op {
     struct AutoBroadcastSpec;
-}
+}   // namespace op
+
+namespace intel_cpu {
```

Comment:
I'm not sure we need to put the shape inference related stuff to the `intel_cpu` namespace. Even though this subsystem is formally housed in the CPU plugin, as far as I know it is meant to be used by all the other plugins as well.

Reply:
All CPU plugin related code should be inside intel cpu namespace. If it is planned to use it in other plugins, it should be moved to a new place, for instance to transformation library.

Reply:
@dmitry-gorokhov , @jane-intel,  could you please express your opinion on this matter?

Reply:
For me it is ok to place shape inference under cpu plugin macros. It doesn't bring any overhead its future extraction.